app.js

```diff
@@ -8,14 +8,23 @@ const {
 } = require("./error-handlers");
 const { getTopics } = require("./controllers/topics-controller");
 const { getApis } = require("./controllers/api-controller");
-const { getArticles } = require("./controllers/articles-controller");
+const {
+  getArticles,
+  getAllArticles,
+} = require("./controllers/articles-controller");
 
 app.get("/api/topics", getTopics);
 
 app.get("/api", getApis);
 
 app.get("/api/articles/:article_id", getArticles);
 
+app.use(express.json());
```

Comment:
This is not necessary for a get request. It is also in a very weird position in your app.

Reply:
<img width="560" alt="Screenshot 2024-08-26 at 13 22 07" src="https://github.com/user-attachments/assets/b4424a7b-fb38-4b29-a4e4-3c59021199ec">
<img width="583" alt="Screenshot 2024-08-26 at 13 22 25" src="https://github.com/user-attachments/assets/b70fb33f-58f9-4165-8aa1-b19eb18e289d">